docs/Running-behind-nginx.md

```diff
@@ -1,8 +1,42 @@
-# Netdata via nginx
+# Running Netdata Behind Nginx #
 
-To pass Netdata via a nginx, use this:
+## Intro ##
 
-### As a virtual host
+[Nginx](https://nginx.org/en/) is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server used to host websites and applications of all sizes. 
+
+The software is known for its low impact on memory resources, high scalability, and its modular, event-driven architecture which can offer secure, predictable performance.
+
+## Why Nginx ##
+
+Nginx is used and useful in cases when you do not want to access Netdata via the default `IP ADDRESS:19999`.
+
+With Nginx, the data is accessed via human-readable domains such as `example.com` or `sub.example.com`. 
+
+By default, Nginx is fast and lightweight out of the box.
+
+Nginx can be configured to secure your Netdata insights by the use of authentication to access the dashboard. 
+
+### Nginx configuration file ###
+
+All Nginx configurations can be found in the `/etc/nginx/` directory. The main configuration file is `/etc/nginx/nginx.conf`. Website or app-specific configurations can be found in the `/etc/nginx/site-available/` directory.
+
+Configuration options in Nginx are known as directives. Directives are organized into groups known as blocks or contexts. The two terms can be used interchangeably.
+
+Depending on your installation source, you’ll find an example configuration file at `/etc/nginx/conf.d/default.conf` or `etc/nginx/sites-enabled/default`, in some cases you may have to manually create the `sites-available` and `sites-enabled` directories. 
+
+You can edit the Nginx configuration file with Nano, Vim or any other Text editors you are comfortable with.
+
+After making changes to the configuration files:
+
+test Nginx configuration with `nginx -t` 
+
+restart Nginx to effect the change with `/etc/init.d/nginx restart` or `service nginx restart`.
+
+## Ways to pass Netdata via Nginx: ##
+
+### As a virtual host ###
+
+With this method instead of `123.45.67.89:19999`, the Netdata dashboard can be accessed via a human-readable URL such as `netdata.example.com` used in the configuration below. 
```

Comment:
There are special addresses for documentation and examples
https://tools.ietf.org/html/rfc5737

> The use of designated address ranges for
   documentation and examples reduces the likelihood of conflicts and
   confusion arising from the use of addresses assigned for some other
   purpose.

>    The blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2),
   and 203.0.113.0/24 (TEST-NET-3) are provided for use in
   documentation.

Maybe we should use them instead of random public addresses.

> inetnum:        123.32.0.0 - 123.47.255.255
netname:        SAMSUNGSDS
descr:          SamsungSDS Inc.

Reply:
@ilyam8 I agree, I have edited the IP address to 192.0.2.0 reserved for documentation.